bin/releasenotes.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+###### Information ############################################################################
+# Name:          releasenotes.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         Create a distro release notes markdown file based on plugin release notes
+#
+# Usage:         ./releasenotes.sh
+#                URLs of plugin release notes are taken from ./releasenotes.txt
+#
+# Platform:      This script works on both GNU/LINUX and MacOS
+#
+# Starting Date: 2020-09-03
+# Modified Date: 2020-09-08
+###############################################################################################
+
+set -e
+# Setup variables
+OLDIFS=$IFS
+REPO_ROOT=`git rev-parse --show-toplevel`
+ROOT=`dirname $(realpath $0)`;
+OD_VERSION=`python $REPO_ROOT/bin/version-info --od`
+ES_VERSION=`python $REPO_ROOT/bin/version-info --es`
+RELEASENOTES_ORIGURL="$ROOT/releasenotes-orig-urls.txt"
+RELEASENOTES_SORTURL="$ROOT/releasenotes-sort-urls.txt"
+RELEASENOTES_TEMPTXT="$ROOT/releasenotes-temp-text.tmp"
+RELEASENOTES_DISTROS="$ROOT/releasenotes-dist-draft.md"
+RELEASENOTES_CATEGORIES=`echo "Breaking Changes,Features,Enhancements,Bug Fixes,Infrastructure,Documentation,Maintenance,Refactoring" | tr '[:lower:]' '[:upper:]'`
```

Comment:
This is a constant...why not just say RELEASENOTES_CATEGORIES="BREAKING CHANGES,FEATURES..." rather than using echo and tr?

Reply:
Sure will do that.